### ISSUE TITLE
Double clicking global connect tool results in error 

### ISSUE THREAD

**Issue by nikku** [bpmn-io/bpmn-js#1402]
__Describe the Bug__

If I by accident double click the palette global connect icon an exception is thrown:

```
TypeError: Cannot read property 'payload' of null
  at move(./node_modules/diagram-js/lib/features/dragging/Dragging.js:183:27)
  at init(./node_modules/diagram-js/lib/features/dragging/Dragging.js:525:7)
  at start(./node_modules/diagram-js/lib/features/global-connect/GlobalConnect.js:99:18)
  at toggle(./node_modules/diagram-js/lib/features/global-connect/GlobalConnect.js:116:15)
  at action(./node_modules/bpmn-js/lib/features/palette/PaletteProvider.js:137:25)
  at trigger(./node_modules/diagram-js/lib/features/palette/Palette.js:302:15)
  at call(./node_modules/diagram-js/lib/features/palette/Palette.js:150:10)
  at HTMLDivElement.<anonymous>(./node_modules/min-dom/dist/index.esm.js:359:10)

```

Logs suggest that this is introduced in our recent release (v8.0.0).

__Steps to Reproduce__

1. double click global connect tool icon in palette
2. __See above stack trace in dev tools__

__Expected Behavior__

Global connect can be toggled via palette without error(s).

__Environment__

Please complete the following information:

 - Browser: Chrome 74
 - OS: Any
 - Library version: v8.0.0

---

Related to [Sentry issue](https://sentry.io/organizations/camunda-modeling/issues/2098483600)

**Comment by pinussilvestrus** [comment]
I investigated this bug a bit.

Summary:

The initial cause of the raised error is that the dragging context got removed before the `move` operation has been started. In previous versions, we don't execute an additional `move` operation, it has been introduced with the latest changes [regarding `autoActivate` for better keyboard activation of the tools](https://github.com/bpmn-io/diagram-js/commit/513753be7a46e173062f9d3f8dd84507874e8de9#diff-2cc890569bf59cbf3b72a077fbee7209e0efb9cc2155cc2e3b1bd5bfa8491424R100).

The fact we're removing the context is a concern of the ToolManager, in which we simply clean up the dragging operation for palette clicks (without reactivating it). This works for toggling the `globalConnect` tool via editor actions, [but not for the palette](https://github.com/bpmn-io/diagram-js/blob/develop/lib/features/tool-manager/ToolManager.js#L94) because no re-instantiating of the dragging operation happens. 

This was no problem in the past since no `move` operation was initiated after toggling the tools. It's also not a problem for the other tools, [since there we simply activate the tools](https://github.com/bpmn-io/bpmn-js/blob/develop/lib/features/palette/PaletteProvider.js#L107), not toggling them.

To conclude I see the following options

1. Simply unify the tool actions in the palette provider, to also only activate the `globalConnect` tool, and not toggling it, to prevent the additional move operation

```js
'global-connect-tool': {
  // ...
  action: {
    click: function(event) {
      globalConnect.start(event);
    }
  }
},
```

2. Doing some extra work and find a way to properly re-initiate the dragging context for the `globalConnect` tool before the `move` event is happening (this would mean higher effort)
